test/functional/wallet_signer.py

```diff
@@ -192,6 +192,12 @@ def test_valid_signer(self):
         assert(res["complete"])
         assert_equal(res["hex"], mock_tx)
 
+        self.log.info('Test sendall using hww1')
+
+        res = hww.sendall(receivers=[{dest:0.5}, hww.getrawchangeaddress()],options={"add_to_wallet": False})
```

Comment:
In 5c0a1b330294d40112af79af19414b913d0dce8c "Add sendall RPC née sweep"

```suggestion
        res = hww.sendall(recipients=[{dest:0.5}, hww.getrawchangeaddress()],options={"add_to_wallet": False})
```

Reply:
Fixed, thank you